platform/android/MapboxGLAndroidSDK/src/cpp/asset_manager_file_source.hpp

```diff
@@ -15,7 +16,7 @@ template <typename T> class Thread;
 
 class AssetManagerFileSource : public FileSource {
 public:
-    AssetManagerFileSource(jni::JNIEnv&, const jni::Object<android::AssetManager>&, const ResourceOptions);
+    AssetManagerFileSource(jni::JNIEnv&, const jni::Object<android::AssetManager>&, const ResourceOptions, const ClientOptions);
```

Comment:
Also here the added argument seems to introduce a breaking change in the android API. Maybe we could use default arguments?

Reply:
Or am I wrong because this is in the src folder and not in the include folder?

Reply:
Yes, this is not public facing as far as I know.